app/models/csv_input.rb

```diff
@@ -1,5 +1,29 @@
 class CsvInput < ActiveRecord::Base
```

Comment:
Consider moving this logic to a module that the `Equipment` class would include. The methodology to add modules to the Rails load path is described here: http://stackoverflow.com/questions/1068558/oo-design-in-rails-where-to-put-stuff